src/utils.cpp

```diff
@@ -683,20 +683,6 @@ bool is_dir(const std::string &path)
   return std_filesystem::is_directory(buf, ec);
 }
 
-bool file_exists_and_ownedby_root(const char *f)
```

Comment:
Why not keep the function and the test?

Reply:
Mostly because this function is only used in one place and the control flow didn't quite work to keep this function around e.g. it's ok to continue if the file doesn't exit but not ok if it does exist and is not owned by root.

Reply:
Right, that makes sense.